code/modules/projectiles/ammo_datums.dm

```diff
@@ -519,6 +519,7 @@ datum/ammo/bullet/revolver/tp44
 	damage = 20
 	penetration = 20
 	sundering = 1.25
+	shell_speed = 3
```

Comment:
undocumented

Reply:
I...actually forgot I even made that change. Anyway, it's been addressed.

Reply:
This seems too much of a buff no? The extra delay itself is really good